### DISCUSSION TITLE
How to exit Yaegi?

### DISCUSSION THREAD

**Discussion by ari-fikri** [traefik/yaegi#1424] [Q&A] [answered]
I've tried Cmd+C, Ctrl+C, exit ↵, quit ↵, but nothing seems to work.
Ctrl+C return with message "context canceled" 

All I can do is kill the terminal session to end it.

Would be helpful if there is a "help" command too.

**Comment by mvertes** [comment, marked as answer]
`Ctrl-D` works.

**Reply by ari-fikri** [reply]
Thanks, it works
